### PR TITLE
improve: farm ux

### PR DESCRIPTION
**Pull request by viet-nv**
- Pools page: prefer get token info from token list instead of get info from subgraph => usecase: miMatic token, marketing want to show it as MAI (token contract symbol is miMatic -> subgraph returns miMatic)
- Farm page: 
  - add warning on my deposit column if user still has liquidity to stake
  - fix: tooltip farm is not started
  - fix: farm apr is blinking

### CODE REVIEW

File: src/components/YieldPools/styleds.tsx [hunk 1]
```diff
@@ -593,6 +593,8 @@ export const ActionButton = styled(ButtonLight)<{ backgroundColor?: string }>`
   background-color: ${({ theme, backgroundColor }) => backgroundColor || theme.primary + '33'};
   width: 28px;
   height: 28px;
+  min-width: 28px;
```
Comment (nguyenhoaidanh) [inline src/components/YieldPools/styleds.tsx:596]: why do we need `min-width 28px` when we already have `width 28px`.

File: src/components/YieldPools/ProMMFarmGroup.tsx [hunk 2]
```diff
@@ -330,15 +332,15 @@ const Row = ({
...
+            {/* farm.startTime > currentTimestamp && (
               <MouseoverTooltip
                 text={'Starting In ' + getFormattedTimeFromSecond(farm.startTime - currentTimestamp)}
                 width="fit-content"
                 placement="top"
               >
                 <Clock size={14} style={{ marginLeft: '6px' }} />
               </MouseoverTooltip>
-            )}
+             ) */}
```
Comment (namgold) [inline src/components/YieldPools/ProMMFarmGroup.tsx:343]: why not remove this

File: src/components/YieldPools/styleds.tsx (hunk 1 above)
Comment (viet-nv) [reply src/components/YieldPools/styleds.tsx]: removed

**Review by namgold — DISMISSED** [review]

File: src/components/YieldPools/ProMMFarmGroup.tsx (hunk 2 above)
Comment (viet-nv) [reply src/components/YieldPools/ProMMFarmGroup.tsx:343]: business requirement. bring back ending in column so this is temporary hide

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**